specification/2.0/README.md

```diff
@@ -935,12 +935,16 @@ The mesh for a skin is defined with vertex attributes that are used in skinning
 }
 ```
 
-The number of joints that influence one vertex is limited to 4 per set, so referenced accessors must have `VEC4` type and following component formats:
+The number of joints that influence one vertex is limited to 4 per set, so referenced accessors must have `VEC4` type and following component types:
 
 * **`JOINTS_0`**: `UNSIGNED_BYTE` or `UNSIGNED_SHORT`
 * **`WEIGHTS_0`**: `FLOAT`, or normalized `UNSIGNED_BYTE`, or normalized `UNSIGNED_SHORT`
 
-The joint weights for each vertex must be non-negative, and normalized to have a linear sum of `1.0`. No joint may have more than one non-zero weight for a given vertex.
+The joint weights for each vertex must be non-negative. No joint may have more than one non-zero weight for a given vertex.
+
+When the weights are stored using `FLOAT` component type, their linear sum must be as close as possible to `1.0` for a given vertex. When the weights are stored using `UNSIGNED_BYTE` or `UNSIGNED_SHORT` component types, their linear sum before normalization must be `255` or `65535` respectively.
+
+> **Implementation Note:** Otherwise, vertices would be deformed very significantly because the weight error would get multiplied by the joint position. So an error of `1/255` in the weight sum is actually unacceptably large.
```

Comment:
This section reads weird with the "Otherwise" in the front as an implementation note. I think you don't need to put implementation note here.

```suggestion
> Without this requirement, vertices would be deformed significantly because the weight error would get multiplied by the joint position. For example, an error of `1/255` in the weight sum would result in an unacceptably large difference in the joint position.
```

Reply:
For the Float case, the "as close as possible to 1", might lead a developer to choose its own arbitrary error tolerance, which might fall within gltf-validator tolerance, or not.

So I would suggest replacing it to "within +-Err to 1" or something like that, where Err is the value used by gltf-validator.

Reply:
There's no way to pick an error tolerance that isn't arbitrary. Also that still wouldn't be enough to fully specify whether it's allowed. There are multiple possible ways you can sum floating point numbers.

Reply:
+1 for Gary's suggested wording. In addition, we could include an implementation note:

> **Implementation Note**: The glTF Validator uses an error tolerance of ____ when testing glTF assets for conformance.

Reply:
@donmccurdy 
This suggestion has been already addressed in the later commit.